content/influxdb3/cloud-dedicated/reference/admin-ui.md

```diff
@@ -1,100 +0,0 @@
----
```

Comment:
This doesn't delete this material, but rather moves it over into the body of the docs.